data/seeds/002_locations.js

```diff
@@ -0,0 +1,34 @@
+const faker = require('faker');
+
+const fakeData = [
+  {
+    id: 1,
+    groomerId: 'ce68mjqz2by3t2peo6i4',
+    businessName: 'Furry Friends Grooming & Care',
+    address: faker.address.streetAddress('###'),
+    email: faker.internet.email(),
+    phoneNumber: faker.phone.phoneNumberFormat(),
+    lat: faker.finance.amount(-90, 90, 6),
+    lng: faker.finance.amount(-180, 180, 6),
+  },
+  {
+    id: 2,
+    groomerId: 'bftqzfqkq3xzj953q64r',
+    businessName: "Gillian's Fine Pet Grooming",
+    address: faker.address.streetAddress('###'),
+    email: faker.internet.email(),
+    phoneNumber: faker.phone.phoneNumberFormat(),
+    lat: faker.finance.amount(-90, 90, 6),
+    lng: faker.finance.amount(-180, 180, 6),
+  },
```

Comment:
Interesting way of limiting the latitude and longitude search.

Reply:
Thank you. I like your suggestion to use `faker.address.latitude()` and `faker.address.longitude()`. That will make it easier to understand what the fake data should represent.